client/src/component/main.module.css

```diff
@@ -0,0 +1,28 @@
+@media screen and (min-width: 768px){
```

Comment:
Awesome job figuring out the dynamic behavior for large differences in window width.